### ISSUE TITLE
Nit: Install on venv

### ISSUE THREAD

**Issue by muunkky** [christian-byrne/audio-separation-nodes-comfyui#14]
Installation documentation works, however it doesn't clarify that you have to activate venv first. This could be frustrating for new users.

**Comment by christian-byrne** [comment]
Good point. [8b5a32c](https://github.com/christian-byrne/audio-separation-nodes-comfyui/commit/8b5a32cf1868600beb3a44f6fde455fe3864eec3)
